src/init/init.ts

```diff
@@ -43,14 +43,17 @@ const templateDescriptions: {[name: string]: string} = {
   'shop': 'The "Shop" Progressive Web App demo',
 };
 
+// TODO: Add Shop "^2.0.0" generator once Polymer 2.0 Shop template is released.
 let shopGenerator = createGithubGenerator({
   owner: 'Polymer',
   repo: 'shop',
+  semverRange: '^1.0.0',
 });
-
+// TODO: Add Shop "^3.0.0" generator once Polymer 2.0 PSK template is released.
```

Comment:
@robdodson Just want to confirm that this in line with what you were thinking for PSK release versioning:

- The current PSK v2.x branch keep Polymer 1.0 with Hybrid elements
- PSK v3.0 and onward get Polymer 2.0

Our plan is to keep this in for the first few versions after Polymer 2.0 release, and then phase it out once appropriate.